addons/calendar/models/calendar_event.py

```diff
@@ -1256,7 +1256,10 @@ def ics_datetime(idate, allday=False):
 
     def _get_customer_description(self):
         """:return (str): The description to include in calendar exports"""
-        return html2plaintext(self.description) if self.description else ''
+        discuss_url = self.videocall_location
+        if hasattr(self, 'appointment_type_id') and self.appointment_type_id:
+            return discuss_url
+        return self.description if self.description else ''
```

Comment:
I don't think this is necessary, as it's handled by the appointment override already.

It should just return the description from the model here.